.github/workflows/sauce.yaml

```diff
@@ -6,12 +6,17 @@ jobs:
   tests-sauce:
     # Don't run on forks. Sauce tests need access to secrets.
     if: github.repository == 'lit/lit'
-
+    
```

Comment:
```suggestion

```

Reply:
I think i got that covered :) there's been some changes so I'll rerequest a review

Reply:
Looks like it's still there.

BTW I use the `"files.trimTrailingWhitespace": true` VSCode setting to do this cleanup automatically on every save.

Reply:
oh interesting!
i thought yaml files required an trailing space
(github gives me an interresting symbol / suggestion without trailing space)